clang/lib/StaticAnalyzer/Core/ProgramState.cpp

```diff
@@ -226,6 +226,20 @@ ProgramStateRef ProgramState::killBinding(Loc LV) const {
   return makeWithStore(newStore);
 }
 
+/// FieldRegions are expected to be wrapped by an ElementRegion as a canonical
+/// representation. See f8643a9b31c4029942f67d4534c9139b45173504 why.
```

Comment:
It seems that in this comment the `FieldRegion` is a mistake and should be changed to `SymbolicRegion`. (But perhaps the comment should be extended to something like "As a canonical representation, SymbolicRegions should be wrapped by ElementRegions before getting a FieldRegion" to mention all three layers of this trickery.)